crates/cxx-qt-gen/src/generator/rust/signals.rs

```diff
@@ -876,294 +871,24 @@ mod tests {
         );
     }
 
-    #[test]
-    fn test_generate_rust_signal_free() {
```

Comment:
It has now had the generation portion moved to `test_generate_rust_signal` and just asserts they generate the same thing, because for some reason we have a vec and non-vec way of generating signals. Might be able to remove some of that code entirely